client/src/main/java/org/apache/celeborn/client/write/DataPushQueue.java

```diff
@@ -106,7 +104,7 @@ public ArrayList<PushTask> takePushTasks() throws IOException, InterruptedExcept
           if (loc != null) {
             Integer oldCapacity = workerCapacity.get(loc.hostAndPushPort());
             if (oldCapacity == null) {
-              oldCapacity = maxInFlightPerWorker - pushState.inflightPushes(loc.hostAndPushPort());
+              oldCapacity = pushState.remainingAllowPushes(loc.hostAndPushPort());
```

Comment:
This changes means that the amount of netty in-flight requests per worker could not be configured, which cause that the user could not control the amount.

Reply:
After this change,
If `PushStrategy`=`SIMPLE`, the logic is still the same as before this pr;
If `PushStrategy`=`SLOWSTART`, then `capacity=currentMaxReqsInFlight-Inflight`;
So it feels like the configuration is still valid.

Reply:
@mcdull-zhang, any update?

Reply:
cc @RexXiong.

Reply:
Hi @mcdull-zhang , I think you are right, `currentMaxReqsInFlight` differs between `SIMPLE` and `SLOWSTART`, and for the latter the `currentMaxReqsInFlight` can change.

This PR looks good to me. cc @AngersZhuuuu could you take a look?